### PR TITLE
task 8

### PR DESCRIPTION
**Pull request by Bokbulbok7**

### CODE REVIEW

File: __tests__/integration.test.js [hunk 1]
```diff
@@ -263,3 +263,94 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {
+  it("PATCH 200: returns the updated article with the incremented votes by the given amount.", () => {
+    const votes = { incVotes: 5 };
```
Comment (popatre) [inline __tests__/integration.test.js]: the ticket asks for `inc_votes` - will cause a bit of confusion on the front end if you've put it as a different key as expected.

File: __tests__/integration.test.js [hunk 2]
```diff
@@ -263,3 +263,94 @@ describe("POST /api/articles/:article_id/comments", () => {
...
+    const votes = { incVotes: "notANumber" };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(votes)
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("Bad request.");
+      });
+  });
+});
```
Comment (popatre) [inline __tests__/integration.test.js:356]: Nice TDD! Just need the inc_votes key to match the ticket please 🙏

File: db/controllers/articlesController.js [hunk 3]
```diff
@@ -50,3 +50,19 @@ exports.postCommentByArticleId = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.patchVotesByArticleId = (req, res, next) => {
+  const { articleId } = req.params;
+  const { incVotes } = req.body;
```
Comment (popatre) [inline db/controllers/articlesController.js]: inc_votes

**Review by popatre — CHANGES_REQUESTED** [review]
Just need the inc_votes key to match the ticket please 🙏

**Review by ghost — APPROVED** [review]
Fab. I know it sounds picky, in front end the ticket will say that you can use ```{inc_votes: 1}``` as an example and we'll all be baffled when you get there and that doesn't work.

**Pull request merged**